rasa/core/policies/ted_policy.py

```diff
@@ -340,6 +342,7 @@ def get_default_config() -> Dict[Text, Any]:
             POLICY_MAX_HISTORY: DEFAULT_MAX_HISTORY,
             # Determines the importance of policies, higher values take precedence
             POLICY_PRIORITY: DEFAULT_POLICY_PRIORITY,
+            USE_GPU: False,
```

Comment:
Lets keep this `True` because CPU being faster than GPU seems more like a bug and this should be set to `False` only as a workaround of this bug :) 
```suggestion
            USE_GPU: True,
```